org.eclipse.scanning.command/src/org/eclipse/scanning/command/factory/GridModelExpresser.java

```diff
@@ -15,13 +15,12 @@
 
 import org.eclipse.dawnsci.analysis.api.roi.IROI;
 import org.eclipse.scanning.api.points.models.GridModel;
-import org.eclipse.scanning.command.PyExpressionNotImplementedException;
 
 class GridModelExpresser extends PyModelExpresser<GridModel> {
 
 	@Override
 	public String pyExpress(GridModel model, Collection<IROI> rois, boolean verbose) throws Exception {
-		
+
 		// TODO Use StringBuilder

```

Comment:
![INFO](https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO') Complete the task associated to this TODO comment. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)